docusaurus/src/theme/DocSidebar.js

```diff
@@ -7,23 +7,25 @@ const WEB_LINKS = [
   ['Mobile Chat Kit', 'https://getstream.io/chat/ux-kit/']
 ]
 
-export default function DocSidebar(props) {
+export default function DocSidebar({ sidebar, ...props }) {
+  const sidebarItems = useMemo(() => sidebar.map(category => ({ ...category, collapsed: false  })));
```

Comment:
can you add `[sidebar]` as second parameter of `useMemo`?

Reply:
Sure! sorry about that 😓